MAINTAINERS_GUIDE.md

```diff
@@ -0,0 +1,91 @@
+Sure, you will get to contribute cool features to the project.
+But most of your time will be spent reviewing, cleaning up, documenting, answering questions, justifying design decisions - while everyone has all the fun!
+But remember - the quality of the maintainers' work is what distinguishes the good projects from the great.
+So please be proud of your work, even the unglamorous parts, and encourage a culture of appreciation and respect for *every* aspect of improving the project - not just the hot new features.
+
+This document is a manual for maintainers old and new.
+It explains what is expected of maintainers, how they should work, and what tools are available to them.
+
+This is a living document - if you see something out of date or missing, please submit a change!
+
+## What are responsibilities of a maintainer?
+
+It is every maintainer's responsibility to:
+
+1) Expose a clear roadmap for improvements for the project.
+2) Deliver prompt feedback and decisions on pull requests.
+3) Be available to anyone with questions, bug reports, criticism etc. on their component.
+   This includes Slack, mailing-list, and GitHub issues and pull requests.
+4) Respect and uphold the philosophy, design, compatibility considerations, and roadmap of the project.
+
+## How are decisions made?
+
+Short answer: with pull requests to this repository.
+
+This is an open-source project with an open design philosophy.
+This means that the repository is the source of truth for EVERY aspect of the project, including its philosophy, design, roadmap and APIs.
+*If it's part of the project, it's in the repo.
+It's in the repo, it's part of the project.*
+
+As a result, all decisions can be expressed as changes to the repository.
+A spec change is a change to the specification.
+An implementation change is a change to the source code.
+A philosophy change is a change to the philosophy manifesto.
+And so on.
+
+All decisions affecting this project, big and small, follow the same 3 steps:
+
+* Step 1: Open a pull request. Anyone can do this.
+
+* Step 2: Discuss the pull request. Anyone can do this.
+
+* Step 3: Accept (`LGTM`) or refuse a pull request. The relevant maintainers do this (see below "Who decides what?")
+
+*I'm a maintainer, should I make pull requests too?*
+
+Yes.
+Nobody should ever push to the `main` branch directly.
+All changes should be made through a pull request.
+
+Bigger changes (that may span projects): an [OCI Working Group](https://github.com/opencontainers/tob/blob/main/CHARTER.md) may be needed.
+
+## Who decides what?
+
+All decisions are pull requests, and the relevant maintainers make decisions by accepting or refusing the pull request.
+Review and acceptance by anyone is denoted by adding a comment in the pull request: `LGTM` (or using the "Approve" feature in GitHub PR reviews).
+However, only currently listed `MAINTAINERS` are counted towards the required **two LGTMs**.
```

Comment:
We often run into situations where it's a maintainer who's opened a PR and it's unclear whether their "self-LGTM" counts towards the two or not -- should we continue to handle these case-by-case, or do any maintainers think it's worth explicitly calling out one way or the other directly in the guide?

Similarly, should we call out "trivial URL fix" exceptions like https://github.com/opencontainers/runtime-spec/pull/1157 explicitly, or continue to handle those case-by-case as well?

(I don't have any strong opinions on either of these one way or another, just things that come to mind while I'm reading the doc.)

Reply:
Is there a way in the github approval process to reduce the LGTM is the PR is from a maintainer?

Reply:
You cannot self-LGTM with GitHub reviews, so if we require GitHub reviews then maintainers already cannot self-LGTM. But we should have an explicit statement whether self-LGTMs are allowed (I've gone back and forth on the issue over the years, but there should be a statement of what the rule is so we're all on the same page).